### ISSUE TITLE
Is it possible to automate a command like get NOTAMs for x airports every day?

### ISSUE THREAD

**Issue by cptalpdeniz** [Siaff/DunDunv2#2]
I want to get NOTAMs for all the airports in Turkey and run it every few days for example (or just post the new ones). Is there way to automate this?

**Comment by Siaff** [comment]
Yeah, it's possible. I don't remember which library I used for NOTAMS, either you recode and brute it by just writing down every airport in Turkey and sending it to the API or you check the libs and see if there's already a function for it! 

Cheers
Siaff
